content/blog/2016/2016-09-30-jenkins-world-2016-wrap-up-community.adoc

```diff
@@ -0,0 +1,75 @@
+
+image:/images/post-images/2016-jenkins-world-wrap/jenkins-world-banner-pic.jpg[Jenkins World 2016,height=200,role=right]
+
+This year's link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World] conference
+was a huge milestone for the Jenkins project - the first global event for the Jenkins community.
+It brought users and contributors together to exchange ideas on the current state
+of the project, celebrate accomplishments of the past year, and look ahead at all the exiting enhancements
+coming down the pipe(line).
+
+=== Contributor Summit
+To kick off Jenkins World, we had a full day "Contributor Summit".
+Jenkins is a distributed project with contributors from all over the globe.
+Conferences like this are perfect time to get contributors together face-to-face,
+to talk through current issues and upcoming plans for the the project.
+Some key topic discussed during this summit were:
+
+* *Infrastructure* - In the past year, the Jenkins project has moved to a new website and has entered a
+link:/blog/2016/05/18/announcing-azure-partnership.adoc[partnership with Microsoft Azure]
+to host our infrastructure.
+* *Events* - A year ago, there were five
+link:/projects/jam/[Jenkins Area Meetups], today there are 37 around the
+world, with ~7000 members.
+* *Security* - Daniel Beck has done a great job a "Security Officer" for the project over the last year.
+Jenkins 2 includes tighter security out of the box, 9 security alerts have been addressed, and the
+link:https://jenkins.io/security/[Security Team] is continuing to evaluate threats as they are reported.
+* *Pipeline* - Pipeline has been a success and there's more goodness in progress, including better
+Pipeline Library support, a UI-based Pipeline Editor, and Declarative Pipeline syntax.
+* *Blue Ocean* - Blue Ocean announced their "1.0 Beta" release and discussed their roadmap.
+* *Storage Pluggability* - One of the big upcoming goals is reducing Jenkins'
+dependence on local file system storage on the server system
+(job configuration, build logs, etc.).  There was extensive
+discussion of how to accomplish this goal.
+
+image:/images/post-images/2016-jenkins-world-wrap/contributors.jpg[Contributors Summit,role=center]
+
+=== Keynote: The State of Jenkins Community
+The next day, gave a great link:https://www.cloudbees.com/jenkins-community-keynote[keynote],
+showing how far the project as come this year and where it is headed.
+You can get the slides
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-jenkins_world_keynote.pdf[here]
+or see the full video below.
+
+++++
+<center>
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/NXjeHeIYRRI" frameborder="0" allowfullscreen></iframe>
+</center>
+++++
+
+
+=== What's Next?
+Overall, Jenkins World was very enjoyable event. I'm sure everyone came away having
+learned a lot and made many new connections.  I know I'm certainly excited to see
```

Comment:
"certainly" seems like an awkward adverb to throw in here, I don't have a good "rule" as to why. Just seems weird to follow "I know".

I think you could drop it.